src/pages/common/programmer_pov.md

```diff
@@ -21,20 +21,20 @@ layout: ../../layouts/MainLayout.astro
 
 </details>
 
-## Polish materials
-
-- [WDI 2021 - Konstanty Orzeszko - Jak straciłem 600 000 PLN w 5 lat? - O tym, jak nie przepalać pieniędzy (na głupoty) jako programista i jak nimi mądrze zarządzać](https://www.youtube.com/watch?v=mGuSdLWV1pk)
-- [Duży w Maluchu - ZABRALI MI MALUCHA! JAK ZARABIAĆ 2500 ZŁ DZIENNIE? \*Patologie branży IT - Ciekawy podcast o branży IT, jak tytuł sugeruje - mówi o różnych patologiach, ale też wspomina o rekrutacji, jak najwięcej zarabiać, problemy z legacy code](https://www.youtube.com/watch?v=aHffnGVK-Vs)
-- [W jakim wieku można zostać seniorem? Zobacz historię Jakuba Pawelskiego, który osiągnął to mając 19 lat](https://geek.justjoin.it/mlody-senior-developer)
-
 ## Career path advices
 
 - [Kenny Gunderman - Should You Learn Native or Cross-Platform Mobile Development?](https://youtu.be/Mq_HS-o-v6o)
 - [Web Dev Junkie - My list of advanced web development topics you should learn](https://www.youtube.com/watch?v=5PWXfjCCm1g)
 - [Code Creative - 3 Skills That Will Help You Succeed as a Junior Developer](https://www.youtube.com/watch?v=I0Erp3w8qu4)
 
-## Valuable tips
+## Valuable tips from pros
 
 - [Ask Cloud Architech - Contributing to Open Source Can Change Your Life - Here’s How to Do It](https://www.youtube.com/watch?v=CML6vfKjQss)
 - [Jack Herrington - Senior Developer Workflow For Stress-Free Coding](https://www.youtube.com/watch?v=eAfUfKYcvBo)
 - [Fireship - Problem-Solving for Developers - A Beginner's Guide](https://www.youtube.com/watch?v=UFc-RPbq8kg)
+
+## Stories from Polish programmers
```

Comment:
The stories could be potentially in English, but all of them are in Polish.

Reply:
This title can be updated as more materials will be added

No need to create universal names for content that doesn't exist yet.